### ISSUE TITLE
White background in mobile browser

### ISSUE THREAD

**Issue by wramalho** [pengx17/logseq-publish#49]
Accessing pages published through logseq-publish (such as my own or pengx17/knowledge-garden) displays a white background if using a mobile browser. Is it an issue with my mobile browser (Chrome, Android 12, Xiaomi POCO f13) or is there another reason for that?

By white background I mean - most custom CSS is displayed, but the color of the background is changed to white.

Best regards,

**Comment by pengx17** [comment]
Does it work on a computer web browser?

**Comment by wramalho** [comment]
Yes! In my computer web browser (both Windows and Linux; Chrome and Librewolf, respectivelly). I will provide screenshots later today
